conans/client/cache/remote_registry.py

```diff
@@ -108,6 +108,8 @@ def __init__(self):
     @classmethod
     def defaults(cls):
         result = Remotes()
+        result._remotes["ConanCenter"] = Remote("ConanCenter", "https://center.conan.io", True,
```

Comment:
how about making a constant, as it's used in many places?

Reply:
Yes, please make it a constant, and make it lowercase, ``-r=conancenter`` will be the best.